docs/source/guides/extensions.rst

```diff
@@ -1,18 +1,17 @@
-Create your own trainer extension
-=================================
+Trainer Extensions
+==================
 
 .. currentmodule:: chainer
 
 In this section, you will learn about the following things:
```

Comment:
Oh, this was not introduced in this PR but let's change _things_ to _topics_ (more formal).